docs/pages/_app.tsx

```diff
@@ -7,10 +7,11 @@ import { palette } from '@ag.ds-next/ag-branding';
 
 const LinkComponent = ({
 	children,
+	className,
 	...props
-}: PropsWithChildren<LinkProps>) => (
+}: PropsWithChildren<LinkProps & { className?: string }>) => (
 	<Link {...props}>
-		<a>{children}</a>
+		<a className={className}>{children}</a>
```

Comment:
This allows `LinkComponent` to be passed in the `as` prop to `Box` and take on Box styling. This pattern should be documented in core.